### DISCUSSION TITLE
How comited are you to integer counts for Gene Expression?

### DISCUSSION THREAD

**Discussion by dagarfield** [GreenleafLab/ArchR#1876] [General]
Or even ATAC for that matter (but let's chat RNA for now). 

There's an increasing number of approaches for mapping RNA reads in the face of uncertainty (e.g. some of the EM options in STAR-solo) that result in non-integer counts. Your notes for `addGeneExpressionMatrix` however, notes:

```
A a scRNA-seq SummarizedExperiment (cell x gene) to be integrated with the scATAC-seq data. Cell names from this object much match those of the cell names in the ArrowFiles/ArchRProject. We will add support shortly for Seurat Objects (see Seurat::as.SingleCellExperiment). The provided values MUST be in counts (integer), not log transformed.
```

Thanks!

**Comment by rcorces** [comment]
I think long-term we arent committed to this. But the current implementation has this expectation.
